.ci/actions/backporting/action.yml

```diff
@@ -37,18 +48,46 @@ runs:
       uses: actions/checkout@v3
       with:
         fetch-depth: 0
+    - name: Setup default values
+      shell: bash
+      env:
+        REQUESTED_REVIEWERS: ${{ toJSON(github.event.pull_request.requested_reviewers) }}
+        MERGED_BY: ${{ github.event.pull_request.merged_by.login }}
+        CUSTOM_REVIEWERS: ""
+      run: |
+        # -- Additional Reviewers --
+        # trick to save multiline string as env variable
+        echo "REQUESTED_REVIEWERS<<EOF" >> $GITHUB_ENV
+        input_include_requested_revs=${{ inputs.include-requested-reviewers }}
+        [[ "$input_include_requested_revs" == "true" ]] && echo "$REQUESTED_REVIEWERS" >> $GITHUB_ENV || echo "[]" >> $GITHUB_ENV
+        echo "EOF" >> $GITHUB_ENV
+
+        # -- Custom Reviewers --
+        input_custom_revs=${{ inputs.custom-reviewers }}
+        [[ ! -z "$input_custom_revs" ]] \
+          && echo "CUSTOM_REVIEWERS=$(echo "[\"${input_custom_revs/,/\",\"}\"]")" >> $GITHUB_ENV \
+          || echo "CUSTOM_REVIEWERS=[]" >> $GITHUB_ENV
+
+        # -- Merged By Reviewer --
+        input_include_merge=${{ inputs.include-merge-as-reviewer }}
+        [[ "$input_include_merge" == "true" ]] && echo "MERGED_BY=$MERGED_BY" >> $GITHUB_ENV || echo "MERGED_BY=" >> $GITHUB_ENV
     - name: Prepare reviewers
       id: prepare-reviewers
       shell: bash
       run: |
-        echo "Additional reviewers retrieved below"
-        echo "${{ inputs.additional-reviewers }}"
-        reviewers="$(echo ${{ inputs.additional-reviewers }} | jq -c 'map(.login)' | jq -cr '. |= . + ["${{ inputs.author }}"] | unique | join(",")')"
+        env
+
```

Comment:
is this required?

Reply:
I added this mainly for debugging purposes, it can help investigations when something goes wrong